playwright/components/split-button/index.ts

```diff
@@ -0,0 +1,22 @@
+import { Page } from "playwright-core";
+import {
+  SPLIT_TOGGLE_BUTTON,
+  ADDITIONAL_BUTTONS,
+  SPLIT_MAIN_BUTTON,
+  MAIN_BUTTON,
+} from "./locators";
+
+// component preview locators
+export const splitToggleButton = (page: Page) =>
```

Comment:
**question** (non-blocking): just double-checking, are any of these used in `carbon-provider.pw.tsx`?

Reply:
It was originally used in the cypress test, so I converted it. Then when I got in to the actual act of converting the tests to Playwright, I found a more reliable way to do what I wanted it to do. I figured someone along the line will need this doing anyway, hence I just left it refactored to playwright.